### PR TITLE
Fix build and refactor

### PR DESCRIPTION
**Pull request by jmle**

### CODE REVIEW

File: java-analyzer-bundle.core/src/main/java/io/konveyor/tackle/core/internal/SymbolInformationTypeRequestor.java
Comment (shawn-hurley) [inline java-analyzer-bundle.core/src/main/java/io/konveyor/tackle/core/internal/SymbolInformationTypeRequestor.java:55]: This is actually kind of a nice way of doing this

File: java-analyzer-bundle.core/src/main/java/io/konveyor/tackle/core/internal/symbol/SymbolProvider.java
Comment (shawn-hurley) [inline java-analyzer-bundle.core/src/main/java/io/konveyor/tackle/core/internal/symbol/SymbolProvider.java:82]: I think that this has to handle the time that the location is null here, it will give everything the ability to that and it would be really nice:

https://github.com/konveyor/java-analyzer-bundle/pull/32/files#diff-56e5b92b0c6a5238cdd5025681448c0bf79a4d3ab0da5e0d43677e33cf53ec65R35-R45


something like that??

Comment (shawn-hurley) [reply java-analyzer-bundle.core/src/main/java/io/konveyor/tackle/core/internal/symbol/SymbolProvider.java:82]: This can be a follow-on though

**Review by shawn-hurley — APPROVED** [review]
Once we get sign-off I would love to merge this, I think that it is a great PR that really helps!

File: java-analyzer-bundle.core/src/main/java/io/konveyor/tackle/core/internal/symbol/SymbolProvider.java
Comment (jmle) [reply java-analyzer-bundle.core/src/main/java/io/konveyor/tackle/core/internal/symbol/SymbolProvider.java:82]: Will do :+1:

**Pull request merged**
